src/components/02-elements/cookie-consent/cookie-consent.njk

```diff
@@ -0,0 +1,39 @@
+<head>
+    <!-- Matomo -->
+    <script type="text/javascript">
+      var _paq = window._paq = window._paq || [];
+      var siteID = null;
+      /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+      _paq.push(['requireCookieConsent']); // Don't use cookies unless we have consent
+      _paq.push(['setCookieDomain', '*.threesixtygiving.org']);
+      _paq.push(['setDomains', '*.threesixtygiving.org']);
+      _paq.push(['setDownloadExtensions', "json|csv|xlsx"]);
+      _paq.push(['trackPageView']);
+      _paq.push(['enableLinkTracking']);
+      (function() {
+        var u="https://analytics.threesixtygiving.org/";
+        _paq.push(['setTrackerUrl', u+'matomo.php']);
+        if(!siteID) throw "siteID is missing";
```

Comment:
It is good practice to use [Error](https://javascript.info/try-catch#throw-operator) if you can because handlers may expect this class/object.

```javascript 
throw new Error("siteID not set");
```